src/Solutions/Folds.idr

```diff
@@ -13,16 +13,12 @@ import Data.Vect
 -- 1
 
 anyList : (a -> Bool) -> List a -> Bool
-anyList p []        = False
-anyList p (x :: xs) = case p x of
-  False => anyList p xs
-  True  => True
+anyList p Nil       = False
```

Comment:
Please do not delete the previous version but give your version a new name. I'll then explain the difference between the two in a separate PR.

Reply:
done: 2b77f31d55888521fb0995730ddd7482715f4e36